content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -101,6 +101,35 @@ To delete a cluster group, select `Delete Cluster form the **Settings** and ente
 * Azure
 * AKS
 * VMWare
+* OpenShift
+
+<Tabs>
+<Tabs.TabPane tab="Configure OpenShift" key="Configure OpenShift">
+
+#### Configure OpenShift 
+<br />
+To deploy a new virtual cluster on OpenShift:
+
+
+1. Select the new Cluster Group and click **Settings**.
```

Comment:
The cluster group doesn't necessarily have to be new. Maybe this should say, `Create a new Cluster Group or edit an existing Cluster Group and click **Settings**.`

Reply:
Updated.